tests/detect_machine.sh

```diff
@@ -2,99 +2,27 @@
-  adecflow01.acorn.wcoss2.ncep.noaa.gov) MACHINE_ID=wcoss2 ;; ### acorn
-  adecflow02.acorn.wcoss2.ncep.noaa.gov) MACHINE_ID=wcoss2 ;; ### acorn
-  dlogin01.dogwood.wcoss2.ncep.noaa.gov) MACHINE_ID=wcoss2 ;; ### dodwood
-  dlogin02.dogwood.wcoss2.ncep.noaa.gov) MACHINE_ID=wcoss2 ;; ### dodwood
-  dlogin03.dogwood.wcoss2.ncep.noaa.gov) MACHINE_ID=wcoss2 ;; ### dodwood
-  dlogin04.dogwood.wcoss2.ncep.noaa.gov) MACHINE_ID=wcoss2 ;; ### dodwood
-  dlogin05.dogwood.wcoss2.ncep.noaa.gov) MACHINE_ID=wcoss2 ;; ### dodwood
-  dlogin06.dogwood.wcoss2.ncep.noaa.gov) MACHINE_ID=wcoss2 ;; ### dodwood
-  dlogin07.dogwood.wcoss2.ncep.noaa.gov) MACHINE_ID=wcoss2 ;; ### dodwood
-  dlogin08.dogwood.wcoss2.ncep.noaa.gov) MACHINE_ID=wcoss2 ;; ### dodwood
-  dlogin09.dogwood.wcoss2.ncep.noaa.gov) MACHINE_ID=wcoss2 ;; ### dodwood
-  clogin01.cactus.wcoss2.ncep.noaa.gov) MACHINE_ID=wcoss2 ;; ### cactus
-  clogin02.cactus.wcoss2.ncep.noaa.gov) MACHINE_ID=wcoss2 ;; ### cactus
-  clogin03.cactus.wcoss2.ncep.noaa.gov) MACHINE_ID=wcoss2 ;; ### cactus
-  clogin04.cactus.wcoss2.ncep.noaa.gov) MACHINE_ID=wcoss2 ;; ### cactus
-  clogin05.cactus.wcoss2.ncep.noaa.gov) MACHINE_ID=wcoss2 ;; ### cactus
-  clogin06.cactus.wcoss2.ncep.noaa.gov) MACHINE_ID=wcoss2 ;; ### cactus
-  clogin07.cactus.wcoss2.ncep.noaa.gov) MACHINE_ID=wcoss2 ;; ### cactus
-  clogin08.cactus.wcoss2.ncep.noaa.gov) MACHINE_ID=wcoss2 ;; ### cactus
-  clogin09.cactus.wcoss2.ncep.noaa.gov) MACHINE_ID=wcoss2 ;; ### cactus
+  alogin0[12].acorn.wcoss2.ncep.noaa.gov)    MACHINE_ID=wcoss2_a ;; ### acorn
+  adecflow0[12].acorn.wcoss2.ncep.noaa.gov)  MACHINE_ID=wcoss2_a ;; ### acorn
+  dlogin0[1-9].dogwood.wcoss2.ncep.noaa.gov) MACHINE_ID=wcoss2 ;; ### dodwood
+  clogin0[1-9].cactus.wcoss2.ncep.noaa.gov)  MACHINE_ID=wcoss2 ;; ### cactus
 
-  gaea9)                   MACHINE_ID=gaea ;; ### gaea9
-  gaea10)                  MACHINE_ID=gaea ;; ### gaea10
-  gaea11)                  MACHINE_ID=gaea ;; ### gaea11
-  gaea12)                  MACHINE_ID=gaea ;; ### gaea12
-  gaea13)                  MACHINE_ID=gaea ;; ### gaea13
-  gaea14)                  MACHINE_ID=gaea ;; ### gaea14
-  gaea15)                  MACHINE_ID=gaea ;; ### gaea15
-  gaea16)                  MACHINE_ID=gaea ;; ### gaea16
-  gaea9.ncrc.gov)          MACHINE_ID=gaea ;; ### gaea9
-  gaea10.ncrc.gov)         MACHINE_ID=gaea ;; ### gaea10
-  gaea11.ncrc.gov)         MACHINE_ID=gaea ;; ### gaea11
-  gaea12.ncrc.gov)         MACHINE_ID=gaea ;; ### gaea12
-  gaea13.ncrc.gov)         MACHINE_ID=gaea ;; ### gaea13
-  gaea14.ncrc.gov)         MACHINE_ID=gaea ;; ### gaea14
-  gaea15.ncrc.gov)         MACHINE_ID=gaea ;; ### gaea15
-  gaea16.ncrc.gov)         MACHINE_ID=gaea ;; ### gaea16
+  gaea5[1-8])                    MACHINE_ID=gaea ;; ### gaea51-58
+  gaea5[1-8].ncrc.gov)           MACHINE_ID=gaea ;; ### gaea51-58
 
-  gaea5[0-9]*)             MACHINE_ID=gaea-c5 ;; ### GAEA C5 login node
+  hfe0[1-8])                   MACHINE_ID=hera_c ;; ### hera01-08
```

Comment:
@FernandoAndrade-NOAA Can you switch hfe05-08 to Rocky8 nodes 'hera' for today's Hera PM?

Reply:
@WenMeng-NOAA updated, thanks for catching that